CONTRIBUTING.md

```diff
@@ -0,0 +1,187 @@
+# Come one, come all, with contributions great and small!
+
+### Let's get you onto our growing list of contributors!
+
+>First off, thank you for considering contributing to the Energy Price News API. It's people like you that make this thing such a great tool.
+
+## Contributing to the project.
+
+Please take a moment to review this document in order to make the contribution
+process easy and effective for everyone involved.
+
+Following these guidelines helps to communicate that you respect the time of
+the developers managing and developing this open source project. In return,
+they should reciprocate that respect in addressing your issue, assessing
+changes, and helping you finalize your pull requests.
+
+As for everything else in the project, the contributions to Energy-Price-News-API are governed by our [Code of Conduct](https://github.com/Energy-Price-News-API/energy-prices-api/blob/master/CODE_OF_CONDUCT.md).
+
+### What kinds of contributions we are looking for.
+
+We keep an open mind! Improving documentation, bug triaging, or writing tutorials are all examples of helpful contributions for both the users of this tool **and** future contributors.
+
+## Your First Contribution
+
+The aim for this project it to be very easy to access, so please have a look through the [issues](https://github.com/Energy-Price-News-API/energy-prices-api/issues) listed to see if any suit your current or desired skills. The maintainers of the project will try to keep it all as up-to-date as possible.
+
+I would like to encourage contributions of any kind to this project:
+
+- Issues:
+  - Suggestions on how to do something
+  - Problems/bugs found in the code
+  - Ideas for features
+  - Discussion around best practices so that others can learn from your experience/knowledge
+- Pull Requests
+  - Should you have a feature you'd like to share already coded, please submit a detailed pull request
+  - A preferred PR format is not yet defined, so we ask you just use common sense and include the type of info you'd like to see as a maintainer
+- Documentation
+  - Spellchecking and typo corrections are **ALWAYS** welcome :joy:, seriously, I hate typos but make them frequently!
+  - Further detail. If you feel something needs to be expanded on, please do so and submit your PR.
+  - Wiki: If we get to a point that a wiki would be helpful, please open an issue/discussion with your suggestions
+  - As the project grows, this `CONTRIBUTING.md` file will need updating, so add to it where you see fit.
+- Share what you do
+  - In order to encourage others to join in, feel free to share what you have done on your social media/websites
+  - If you know someone that has done something similar, please show them this project as their input would be greatly appreciated
+
+### A link to a resource for people who have never contributed to open source before.
```

Comment:
Maybe we can drop this resource section for now and if someone struggle to make a contribution, we give it this resource to get started :thinking:

Reply:
done!